packages/stable/src/__tests__/api/filesMultiPart.int.spec.ts

```diff
@@ -1,5 +1,3 @@
-// Copyright 2020 Cognite AS
```

Comment:
I see the copyright is kept in some files, and removed in some.
Did you intend to delete the copyright comment?

Reply:
not sure but added back